Dockerfile

```diff
@@ -93,19 +93,63 @@ RUN pip install -e . --no-deps
 ###################
 ## Frontend Base ##
 ###################
-FROM node:16-slim as frontend
+FROM node:16-alpine as frontend
+
+RUN apk add --no-cache libc6-compat
+# Build the frontend clients
+WORKDIR /fides/clients
+COPY clients/package.json ./
+COPY clients/package-lock.json ./
+COPY clients/fides-consent/package.json ./fides-consent/package.json
+COPY clients/admin-ui/package.json ./admin-ui/package.json
+COPY clients/privacy-center/package.json ./privacy-center/package.json
 
-# Build the admin-ui frontend
-WORKDIR /fides/clients/admin-ui
-COPY clients/admin-ui/package.json clients/admin-ui/package-lock.json ./
 RUN npm install
-COPY clients/admin-ui/ .
+
+COPY clients/ .
 
 ####################
 ## Built frontend ##
 ####################
 FROM frontend as built_frontend
-RUN npm run export
+
+# Builds and exports admin-ui
+RUN npm run export-ui
+# Builds privacy-center
+RUN npm run build-pc
+
+#############################
+## Production PC ##
+#############################
+FROM node:16-alpine as prod_pc
+
+WORKDIR /fides/clients
+
+ENV NODE_ENV production
+ENV NEXT_TELEMETRY_DISABLED 1
+
+RUN addgroup --system --gid 1001 nodejs
+RUN adduser --system --uid 1001 nextjs
+
+RUN chown nextjs:nodejs /fides/clients
+
+# We need to copy everything so we can rebuild with the new configs if needed
+COPY --from=built_frontend /fides/clients /fides/clients
```

Comment:
I really want to get rid of this rebuild logic! Going to poke at it this ASAP... 😄